docs/app-guide/add-a-onepassword-secret.rst

```diff
@@ -0,0 +1,116 @@
+###########################################
+Add a secret with 1Password and VaultSecret
+###########################################
+
+Whenever possible, static secrets for applications are stored in a 1Password vault before being automatically synced to the Vault service itself and ultimately to Kubernetes ``Secret`` resources via :ref:`vault-secrets-operator`.
+For an overview of secrets architecture, see :ref:`secrets`.
+
+This page provides steps for adding an application secret through 1Password.
+
+Part 1. Open the 1Password vault
+================================
+
+In one password, access the **LSST IT** 1Password team and open the vault named ``RSP-Vault``.
+Items in this vault are synced into Kubernetes ``Secret`` resources.
+
+Part 2. Create a Secret Note
+============================
+
+Each item in a Kubernetes ``Secret`` corresponds to the contents of a secret note in 1Password.
```

Comment:
Either a secure note (1Password calls them that, not secret note) or a login can be used (I prefer logins). Both work the same way.

Reply:
So when using a login item, only the password field is used? The username part is optional/ignored? I couldn't figure out how this worked, so I didn't want to document it.

Reply:
Correct, when using a login item only the password is used and the username is ignored (and can be used for documentation purposes).